src/inttest/resources/config/application.yml

```diff
@@ -53,6 +53,9 @@ faf-api:
     max-size-bytes: 4096
     image-width: 40
     image-height: 20
+  cloudflare:
+    hmac-secret: "banana"
+    hmac-param: "verify"
```

Comment:
Is this actually configurable? If it is fixed by cloudflare I wouldn't make it configurable.

Reply:
Yes it is configurable